openmdao/docs/features/recording/case_reading.rst

```diff
@@ -44,7 +44,7 @@ The `get_case` method provides a way to get at individual cases. The argument to
             as is normally done in Python.
     #. String - in which case the argument is one of the case keys.
 
-For example, in the common situation where the user wants to see the last case, they can do
+For example, in the common situation where the user wants to see the last case, they can do:
```

Comment:
For future consideration... "the user" is actually the reader, so it may be awkward reading about oneself this way (and as "they").    I'm no English/Communications expert, but maybe using the pronoun "You" is preferable?